modules/ocf/manifests/hidepid.pp

```diff
@@ -0,0 +1,36 @@
+class ocf::hidepid {
+
+  $group_cap_ptrace = 'capptrace'
+
+  group { $group_cap_ptrace:
+    ensure => present,
+    name   => $group_cap_ptrace,
+    system => true,
+    before => Mount['/proc'];
+  } ->
+
+  ocf::systemd::override { 'hidepid-logind':
+    unit    => 'systemd-logind.service',
+    content => "[Service]\nSupplementaryGroups=${group_cap_ptrace}\n",
+  }
+
+  if str2bool($::polkit_installed) {
+    # NOTE: Futureproof for policykit for version 0.115. Since polkit < 0.115
+    # runs as root this change incurs no security cost.
+    ocf::systemd::override { 'hidepid-polkit':
```

Comment:
Also: according to that Debian wiki page, since polkitd itself does the privilege drop, adding a supplementary group doesn't help. Adding the `polkitd` user to `$group_cap_ptrace` might though.

Reply:
Hmm... That would work. What about adding `AmbientCapabilities=CAP_SYS_PTRACE` to the unit?